includes/namespace.php

```diff
@@ -80,6 +80,8 @@ function deactivate( $network_wide ) {
 register_activation_hook( WEBSTORIES_PLUGIN_FILE, '\Google\Web_Stories\activate' );
 register_deactivation_hook( WEBSTORIES_PLUGIN_FILE, '\Google\Web_Stories\deactivate' );
 
+// WEB-STORIES-DEV-CODE.
```

Comment:
This is the important bit this PR introduces

Reply:
@spacedmonkey WDYT about this?

Reply:
Why are we removing this code in the first place? Is there really a problem with shipping this into production?